i18n/en.json

```
@@ -1352,6 +1352,10 @@
     "id": "api.file.test_connection.s3.connection.app_error",
     "translation": "Bad connection to S3 or minio."
   },
+  {
+    "id": "api.file.upload_file.incorrect_channelId.app_error",
+    "translation": "Unable to upload file(s). Incorrect ChannelId: {{.channelId}}"
```

Another nit: UploadFile only allows for uploading one file, so we can make this always singular.
```suggestion
    "translation": "Unable to upload the file. Incorrect channel ID: {{.channelId}}"
```